README.md

```diff
@@ -52,13 +52,23 @@ Start the container
       --ipc=host \
       --user=root \
       --privileged \
-      -v /:/host:Z \
-      -it kubevirt/libvirt:latest
+      --volume=/:/host:Z \
+      --volume=/dev:/host-dev \
+      --volume=/var/run/libvirt:/var/run/libvirt \
+      --volume=/sys:/host-sys \
```

Comment:
Ultimately a question of preference, but I think this would look nicer as

```
--volume=/:/host:Z \
--volume=/dev:/host-dev \
--volume=/sys:/host-sys \
--volume=/var/run/libvirt:/var/run/libvirt \
```

eg. with the generic system part first and the libvirt-specific part afterwards.